### PR TITLE
[QUARKS-88] [WIP] Add Apache licence headers to source files

### PR DESCRIPTION
**Pull request by Cazen**
I've checked with this command.
```
Cazens-MacBook-Pro:incubator-quarks Cazen$ find . -not -path '*/\.*' \
> -not -name "*.jar"   \
> -not -name "*.class" \
> -not -name "*.iml"   \
> -not -name "NOTICE"  \
> -not -name "README"  \
> -not -name "*.md"    \
> -not -name "*.json"  \
> -not -name "*.css"   \
> -not -name "*.png"   \
> -not -name "*.js"    \
> | xargs grep -L "http://www.apache.org/licenses/LICENSE-2.0" 2> /dev/null
./analytics/math3/ext/commons-math3-3.4.1/NOTICE.txt
./connectors/javax.websocket-client/src/main/resources/META-INF/services/quarks.javax.websocket.QuarksSslContainerProvider
./connectors/wsclient-javax.websocket/src/test/keystores/clientKeyStore.jks
./connectors/wsclient-javax.websocket/src/test/keystores/clientTrustStore.jks
./connectors/wsclient-javax.websocket/src/test/keystores/serverKeyStore.jks
./connectors/wsclient-javax.websocket/src/test/keystores/serverTrustStore.jks
./console/servlets/webapp_content/html/index.html
./console/servlets/webapp_content/js/ext/d3.BSD.LICENSE
./console/servlets/webapp_content/js/ext/jquery-ui-1.11.4.custom/jquery-ui.LICENSE
./console/servlets/webapp_content/js/ext/jquery-ui-1.11.4.custom/jquery.LICENSE
./console/servlets/webapp_content/js/ext/sankey.d3.BSD.LICENSE
./console/servlets/webapp_content/WEB-INF/console.xml
./ext/slf4j-1.7.12/LICENSE.txt
./KEYS
./quarks_overview.html
./samples/apps/src/main/java/quarks/samples/apps/applicationTemplate.properties
./scripts/apps/sensorAnalytics/sensorAnalytics.properties
./scripts/connectors/iotf/device.cfg
./scripts/connectors/jdbc/jdbc.properties
./scripts/connectors/jdbc/persondata.txt
./scripts/connectors/kafka/kafka.properties
./scripts/connectors/mqtt/mqtt.properties
Cazens-MacBook-Pro:incubator-quarks Cazen$
```

### CODE REVIEW

**Comment by ddebrunner** [comment]
Nice, just to note the list of files in the description is the files without the licence header, I initially thought it was the files that were changed.

**Comment by ddebrunner** [comment]
In the list without the only one I would expect to have the header is:
`quarks_overview.html`

File: spi/topology/src/main/java/quarks/topology/spi/graph/TWindowTimeImpl.java [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
 
-import static quarks.window.Policies.alwaysInsert;
```
Comment (ddebrunner) [inline spi/topology/src/main/java/quarks/topology/spi/graph/TWindowTimeImpl.java]: Any reason these were moved?

Eclipse puts the static imports first, but now they are after the regular imports.

**Comment by Cazen** [comment]
Good Day @ddebrunner Thank you for detail review.

- Yes. The list above are files that does not include Apache License
- I thought "quarks_overview.html" file is snippet style(It start with just BODY tag. Not HTML tag) to assemble into another HEADER(license syntax included) file which explained in http://www.apache.org/legal/src-headers.html#faq-exceptions . But, I will create a new Jira Issue soon because it needs to be include.
- I'm sorry. I didn't mean to change it. IntelliJ automatically adjust the order of import. I'll place back to previous version with above Jira issue.

Thank you :)

**Pull request merged**